mkosi/__init__.py

```diff
@@ -4386,23 +4386,26 @@ def generate_key_cert_pair(args: Args) -> None:
 
 def bump_image_version() -> None:
     """Write current image version plus one to mkosi.version"""
-    version = Path("mkosi.version").read_text().strip()
+
+    version_file = Path("mkosi.version")
+    if not version_file.exists():
+        die(f"Cannot bump image version, '{version_file}' not found")
+
+    version = version_file.read_text().strip()
     v = version.split(".")
 
     try:
-        m = int(v[-1])
+        v[-1] = str(int(v[-1]) + 1)
+        logging.info("Increasing last component of version by one")
```

Comment:
This is excessive, I'd drop it

Reply:
Am I correct in assuming you mean the log message and not the commit?